src/System.Windows.Forms/src/System/Windows/Forms/TreeView.cs

```diff
@@ -1784,7 +1784,7 @@ internal void SetToolTip(ToolTip toolTip, string toolTipText)
         {
             if (toolTip != null)
             {
-                UnsafeNativeMethods.SendMessage(new HandleRef(toolTip, toolTip.Handle), NativeMethods.TTM_SETMAXTIPWIDTH, 0, SystemInformation.MaxWindowTrackSize.Width);
+                User32.SendMessageW(toolTip, WindowMessages.TTM_SETMAXTIPWIDTH, (IntPtr)0, (IntPtr)SystemInformation.MaxWindowTrackSize.Width);
```

Comment:
for consistency sake:
```suggestion
                User32.SendMessageW(toolTip, WindowMessages.TTM_SETMAXTIPWIDTH, IntPtr.Zero, (IntPtr)SystemInformation.MaxWindowTrackSize.Width);
```